### PR TITLE
Update advanced-hunting-devicealertevents-table.md

### PR DESCRIPTION
**Pull request by msbemba**
DeviceAlertEvents No Longer Provided and is replaced by The AlertInfo and AlertEvidence 
Reference https://docs.microsoft.com/en-us/microsoft-365/security/defender/advanced-hunting-migrate-from-mde?view=o365-worldwide

Added an important note at the top of the article and linked the mapping table

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9316

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/advanced-hunting-devicealertevents-table.md [hunk 1]
```diff
@@ -24,7 +24,11 @@ ms.technology: mde
 **Applies to:**
 - [Microsoft Defender for Endpoint Plan 2](https://go.microsoft.com/fwlink/p/?linkid=2154037)
 
-> Want to experience Defender for Endpoint? [Sign up for a free trial.](https://signup.microsoft.com/create-account/signup?products=7f379fee-c4f9-4278-b0a1-e4c8c2fcdf7e&ru=https://aka.ms/MDEp2OpenTrial?ocid=docs-wdatp-advancedhuntingref-abovefoldlink)
+>[!IMPORTANT]
+>The `AlertInfo` and `AlertEvidence` tables replace the `DeviceAlertEvents` table in the Microsoft Defender for Endpoint schema. See more, for
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/advanced-hunting-devicealertevents-table.md]:
```suggestion
> [!IMPORTANT]
> The `AlertInfo` and `AlertEvidence` tables replace the `DeviceAlertEvents` table in the Microsoft Defender for Endpoint schema. To learn more, see [Map DeviceAlertEvents Table](/microsoft-365/defender/advanced-hunting-migrate-from-mde?view=o365-worldwide#map-devicealertevents-table).
```

File: microsoft-365/security/defender-endpoint/advanced-hunting-devicealertevents-table.md [hunk 2]
```diff
@@ -24,7 +24,11 @@ ms.technology: mde
 **Applies to:**
 - [Microsoft Defender for Endpoint Plan 2](https://go.microsoft.com/fwlink/p/?linkid=2154037)
 
-> Want to experience Defender for Endpoint? [Sign up for a free trial.](https://signup.microsoft.com/create-account/signup?products=7f379fee-c4f9-4278-b0a1-e4c8c2fcdf7e&ru=https://aka.ms/MDEp2OpenTrial?ocid=docs-wdatp-advancedhuntingref-abovefoldlink)
+>[!IMPORTANT]
+>The `AlertInfo` and `AlertEvidence` tables replace the `DeviceAlertEvents` table in the Microsoft Defender for Endpoint schema. See more, for 
+[Map DeviceAlertEvents](/defender/advanced-hunting-migrate-from-mde?view=o365-worldwide#map-devicealertevents-table)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/advanced-hunting-devicealertevents-table.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

File: microsoft-365/security/defender-endpoint/advanced-hunting-devicealertevents-table.md [hunk 3]
```diff
@@ -24,7 +24,10 @@ ms.technology: mde
 **Applies to:**
 - [Microsoft Defender for Endpoint Plan 2](https://go.microsoft.com/fwlink/p/?linkid=2154037)
 
-> Want to experience Defender for Endpoint? [Sign up for a free trial.](https://signup.microsoft.com/create-account/signup?products=7f379fee-c4f9-4278-b0a1-e4c8c2fcdf7e&ru=https://aka.ms/MDEp2OpenTrial?ocid=docs-wdatp-advancedhuntingref-abovefoldlink)
+> [!IMPORTANT]
+> The `AlertInfo` and `AlertEvidence` tables replace the `DeviceAlertEvents` table in the Microsoft Defender for Endpoint schema. To learn more, see [Map DeviceAlertEvents Table](/microsoft-365/defender/advanced-hunting-migrate-from-mde?view=o365-worldwide#map-devicealertevents-table).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/advanced-hunting-devicealertevents-table.md:28]:
```suggestion
> The `AlertInfo` and `AlertEvidence` tables replace the `DeviceAlertEvents` table in the Microsoft Defender for Endpoint schema. To learn more, see [Map DeviceAlertEvents Table](/microsoft-365/defender/advanced-hunting-migrate-from-mde?view=o365-worldwide#map-devicealertevents-table).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Kindly guide.

**Comment by tiburd** [comment]
close > reopen

**Comment by msbemba** [comment]
Thank you @tiburd

**Comment by tiburd** [comment]
@msbemba 
This PR can be merged because the validation status = passed.

However, because the suggestion does not affect the technical accuracy, do you want to fix it? Will the bookmark (#map-devicealters-table) work if you remove the view parameter (?view=o365-worldwide)?

**Comment by msbemba** [comment]
> @msbemba This PR can be merged because the validation status = passed.
> 
> However, because the suggestion does not affect the technical accuracy, do you want to fix it? Will the bookmark (#map-devicealters-table) work if you remove the view parameter (?view=o365-worldwide)?

Yes, @tiburd  I just fixed that part of the URL.

**Comment by tiburd** [comment]
@msbemba - thanks!
@yogkumgit - merging :)

**Comment by tiburd** [comment]
reviewed + checked links. merging.

**Pull request merged**